conf/httpd.conf.d/httpd.admin

```diff
@@ -100,8 +100,8 @@ if (defined($management_network->{'Tip'}) && $management_network->{'Tip'} ne '')
     $vhost = "0.0.0.0:".$PfConfig->{'ports'}{'admin'};
 }
 
-$User = "pf";
-$Group = "pf";
+$User = "pf_admin";
+$Group = "pf_admin";
```

Comment:
Because we need a user with all admin rights in the admin interface. The amount of commands we need sudo rights for is increasing a lot